src/lang/modifyAst.test.ts

```diff
@@ -666,189 +660,6 @@ ${!replace1 ? `  |> ${line}\n` : ''}  |> angledLine(angle = -65, length = ${
   })
 })
 
-describe('Testing removeSingleConstraintInfo', () => {
-  describe('with mostly object notation', () => {
-    const code = `part001 = startSketchOn(-XZ)
-  |> startProfile(at = [0, 0])
-  |> line(end = [3 + 0, 4 + 0])
```

Comment:
Lots of these tests were already redundant "with object notation" makes so sense now that everything is kwargs, and in fact you can see that all of these snippets are kwargs.